data/discovery/register/fire-authority.yaml

```diff
@@ -0,0 +1,9 @@
+fields:
+- fire-authority
+- name
+- start-date
+- end-date
+phase: discovery
+register: fire-authority
+registry: home-office
+text: A register of fire authoritites in the England
```

Comment:
`A register of fire authorities in England`

Reply:
i knew that :P